apps/prairielearn/src/cron/autoFinishExams.ts

```diff
@@ -43,6 +43,7 @@ export async function run() {
       await gradeAssessmentInstance(
         assessment_instance.id,
         authn_user_id,
+        authn_user_id,
```

Comment:
As noted in the commented code above, grading is done by the system so users are `null`. I reused `authn_user_id` here rather than storing another `null` variable. However, reviewing now, this looks confusing.

Reply:
Yeah, consider introducing a `user_id = null` variable to make things a little clearer.

Reply:
Added a `user_id = null` in [bc258cb](https://github.com/PrairieLearn/PrairieLearn/pull/10840/commits/bc258cb63105129c9382b9367281d7b040a14698).